gemma/gemma.h

```diff
@@ -112,6 +112,8 @@ class Gemma {
   const ByteStorageT& Prefill() const { return prefill_u8_; }
   const ByteStorageT& Decode() const { return decode_u8_; }
 
+  ByteStorageT& MutableWeights() { return weights_u8_; }
```

Comment:
This would also allow someone to change the pointer itself, which would be strange.
Do we really require a non-const& ? It seems get() can return non-const even if the ptr class itself is const: https://gcc.godbolt.org/z/bv18PKvjM

Reply:
yes, it works with const reference as well!